app/config/Config.scala

```diff
@@ -83,6 +83,8 @@ object Config extends AwsInstanceTags {
   val capiPreviewIAMUrl = config.getString("capi.previewIAMUrl")
   val capiLiveUrl = config.getString("capi.liveUrl")
 
+  val capiApiKey = getOptionalProperty("capi.apiKey", config.getString).getOrElse("atom-workshop-DEV")
```

Comment:
are we happy with this key being in source control?

Reply:
no 😭 sorry about that, will fix.

Reply:
actually, no. I've demoted it to a developer key and added an internal key to AW CODE.